cmd/lightclient/sentinel/communication/ssz_snappy/stream.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/libp2p/go-libp2p/core/network"
 )
 
+var expectedForkDigest = [4]byte{74, 38, 197, 139}
```

Comment:
only expected from mainnet, please make it work for all chain

Reply:
removed and added a todo to check that the fork digest matches.